Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -9132,12 +9132,22 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Returns the internal name of the objective.",
 				},
+				GetPlayers =
+				{
+					Returns =
+					{
+						{
+							Type = "table",
+						},
+					},
+					Notes = "Returns all of the players this objective is tracking",
```

Comment:
Players, as in `cPlayer` instances, or playernames, or UUIDs? Array-table, or a map?